### PR TITLE
Rebase batch of misc commits

### PR DESCRIPTION
**Pull request by jewertow**

### CODE REVIEW

**Comment by dgn** [comment]
TLS changes are in here as well?

**Comment by jewertow** [comment]
> TLS changes are in here as well?

I rebased branch, so those changes are no longer in this PR.

File: bin/update_maistra_deps.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+UPDATE_BRANCH=${UPDATE_BRANCH:-"maistra-2.1"}
```
Comment (jwendell) [inline bin/update_maistra_deps.sh]: This should probably change to something else

**Review by jwendell — APPROVED** [review]

File: bin/update_maistra_deps.sh (hunk 1 above)
Comment (jewertow) [reply bin/update_maistra_deps.sh]: Thanks for finding this! I changed this variable to maistra-2.2.

**Pull request merged**